src/components/sections/TransparencySection.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Transparency from '../Transparency';
+
+const TransparencySection = (props) => (
+	<div className="dg_section dark">
+		<div className="container">
+			<div className="row">
```

Comment:
@sgrg1 wherever this row ends up being you can add a class of `text-center` to align the icon buttons as you wish. I'm not sure where that will be quite yet, but I think you get the point.